dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -ex
+
+while getopts v:a:j:o: flag
+do
+    case "${flag}" in
+        v) version=${OPTARG};;
+        a) version_alias=${OPTARG};;
+        j) json_file=${OPTARG};;
+        o) operation=${OPTARG};;
+        *) echo "ERROR: invalid parameter ${flag}" ;;
+    esac
+done
+
+
+setup(){
+    mkdir /home/airflow/workspace
+    cd /home/airflow/workspace
+    export PATH=$PATH:/usr/bin
+    export AWS_REGION=us-west-2
+    export AWS_ACCESS_KEY_ID=$(cat ${json_file} | jq -r .aws_access_key_id)
+    export AWS_SECRET_ACCESS_KEY=$(cat ${json_file} | jq -r .aws_secret_access_key)
+    export ROSA_TOKEN=$(cat ${json_file} | jq -r .rosa_creds_token)
+    export CLUSTER_NAME=$(cat ${json_file} | jq -r .openshift_cluster_name)
+    rosa login --env=staging
```

Comment:
can we make the env string a variable? It would allow us more flexibility in the future

Reply:
yep, Im going to add as param. and also save a token also for prod

Reply:
added tokens for prod and stage, and make environment a var for each release